File: Data Structures & Algorithms/concatenation-of-array/submission-0.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        var len = 2 * nums.Length;
        var ans = new int[len];
        var i = 0;
        while (i < len) {
            if (i < nums.Length) {
                ans[i] = nums[i];
            } else {
                ans[i] = nums[i- nums.Length];
            }
            i++;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.js
class Solution {
    /**
     * @param {number[]} arr
     * @return {number[]}
     */
    replaceElements(arr) {
        let max = -1;
        let ans = new Array(arr.length);
        for(let i = arr.length-1; i >= 0; i--){
            ans[i] = max;
            max = Math.max(max, arr[i]);
        }
        return ans;
    }
}
